tutor/commands/jobs.py

```diff
@@ -308,12 +309,72 @@ def sqlshell(args: list[str]) -> t.Iterable[tuple[str, str]]:
     Extra arguments will be passed to the `mysql` command verbatim. For instance, to
     show tables from the "openedx" database, run `do sqlshell openedx -e 'show tables'`.
     """
-    command = "mysql --user={{ MYSQL_ROOT_USERNAME }} --password={{ MYSQL_ROOT_PASSWORD }} --host={{ MYSQL_HOST }} --port={{ MYSQL_PORT }} --default-character-set=utf8mb3"
+    command = "mysql --user={{ MYSQL_ROOT_USERNAME }} --password={{ MYSQL_ROOT_PASSWORD }} --host={{ MYSQL_HOST }} --port={{ MYSQL_PORT }} --default-character-set=utf8mb4"
     if args:
         command += " " + shlex.join(args)  # pylint: disable=protected-access
     yield ("lms", command)
 
 
+   
+@click.command(
+    short_help="Upgrade mysql to a specific charset and collation",
+    help=(
+        "Upgrade mysql to a specific charset and collation. You can either upgrade all tables, specify only certain tables to upgrade or specify certain tables to exclude from the upgrade process"
```

Comment:
This docstring is misleading. It implies that it allows the user to upgrade to just any charset/collation. Instead, I suggest that we rename this command to something like `convert-mysql-utf8mb4-charset`.

Reply:
The original command was meant to allow users to upgrade to any charset/collation, however that caused issues with the connection string. However, since we have removed that option, I think the `convert-mysql-utf8mb4-charset` makes more sense. I'll update it.